docs/finalproject/prep.md

```diff
@@ -4,9 +4,35 @@ title: Final Project Preparation
 sidebar_label: Preparation
 ---
 
+## Checklist to organise Final Projects
+Getting the Final Projects ready is not a task that can be done a week before the module. A lot of decisions have to be made beforehand, so this high-level list of the steps Volunteers and PROM (Programme Managers) should do in order to get everything ready for Final Projects. It also has the main links of important documents. 
+
+**When you are on the React Module**
+- [ ] Must review the trainee’s performance and make [readiness for Final Projects decision](https://docs.google.com/document/d/1jMxqI0L7IKFENCQ8Lw-D1lhnj3c3RZI_WK808wG4YJM/edit?usp=sharing)
+- [ ] Identify trainees that will be filling in roles such as UI/UX Designer, Tester or Product Delivery
+- [ ] Ask internally and externally for possible ideas of charities that could be part of Final Projects. The form used should be [this](https://forms.gle/Gn85Kdna6QB2Qfcp9). 
+- [ ] Start recruiting Tech Lead, Product Managers and UI/UX Designer for Final Projects. See the role descriptions [here](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles). 
+
+**When you are on the NodeJS Module**
+- [ ] Review the project ideas with Tech Leads and Product Managers defining technical feasibility and scope size. You might have to have volunteers clarify some information with the charities
+- [ ] Identify which of the Ideas will be used as a final project. 
+	Send feedback to all charities if they have been chosen or not. 
+	For the chosen ones, clarify the [role](https://docs.codeyourfuture.io/volunteers/teams-1/cyf-products-final-projects/roles/product-owner) and who will be the Product Owner from their Charity. 
+- [ ] Onboard the volunteers with the standard process
+- [ ] Use [this template](https://docs.google.com/spreadsheets/d/16vSSJgzCZJKF-2pwuBTkKjJJJ9i1CGRqMbYB-HEO5mo/edit?usp=sharing) to identify the projects that will be used and who will be on which team
+- [ ] Let volunteers choose which project they would like to be on
+
+**When you are on SQL Module**
+- [ ] Trainees must have finalised the Full Stack assessment 
```

Comment:
Hm, maybe something closer to "Trainees must have completed up to Level 300 by week 3 of the module before Final Projects. " ?

Here's some more info about this that might help us rephrase this to get this a little clearer.  They have three weeks to work on the Full Stack Project Assessment.

https://github.com/CodeYourFuture/Full-Stack-Project-Assessment 

Each week has a mandatory minimum level they have to achieve and someone needs to check this has happened. There is a week 4 in the DB module that volunteers have "spare" to pick up any dropped threads on this but trainees have to be at level 300 by week 3.

There are also various optional stretch levels, but doing random parts of the stretch tasks and not completing the core task does not comprise a pass, alas. Levels 100, 200, 250, and 300 must be complete to enter Final Projects in a full stack dev role.